### ISSUE TITLE
Wrong tools location on Git Staging tab

### ISSUE THREAD

**Issue by mstup** [eclipse-egit/egit#49]
### Version

6.10

### Operating System

Linux/Unix

### Eclipse version

2024-06

### Bug description

Tools icons on the Git Staging tab located not as on the remaining tabs. It waste of extra space on tab area.

### Actual behavior

Tools are located in content of the tab on the right.
![image](https://github.com/user-attachments/assets/2d74c6d7-b7a0-4996-9ab5-22d5749c0628)

### Expected behavior

I expected to see all tools on the one line with tabs, as it is on other tabs.
![image](https://github.com/user-attachments/assets/b0001418-3818-4839-ab38-62fe6e9bf247)

### Other information

OS: Fedora 38

**Comment by Bananeweizen** [comment]
That works fine for me on Windows. The tool icons are moved to the tab bar _only_ when they fit into the available free width. I suspect the "filter files" combo to have some layout issue specifically on your OS (I've seen a lot of bad SWT layout issues on GTK when I used that in the past). It might be something like the composite "behind" the input box having a much larger width than the input box itself.

**Comment by mstup** [comment]
@Bananeweizen Yes, it could be the reason but I no way to hide this Search text field to test this assumption.

**Comment by vogella** [comment]
Should be fixed with https://github.com/eclipse-platform/eclipse.platform.swt/pull/3139
